scripts/main.js

```diff
@@ -17,6 +19,7 @@ var result_display = `
     </table>
 `;
 
+// TODO: externalize
 var warmly_btn_label = "Warm";
 
 function refresh(f) {
```

Comment:
Suggestion: We should do something like:
`document.onreadystatechange = function () {
  if (document.readyState === "complete") {
    initApplication();
  }
}`

Source: https://developer.mozilla.org/en-US/docs/Web/API/Document/readyState

Reply:
Suggestion: We should rename f to warmlyStartUpMethod or something like that